package.json

```diff
@@ -29,8 +29,6 @@
   "devDependencies": {
     "@babel/core": "^7.20.5",
     "@emotion/css": "^11.10.6",
-    "@parcel/packager-ts": "^2.8.2",
-    "@parcel/transformer-typescript-types": "^2.8.2",
```

Comment:
There are many more dependencies and dev dependencies that should be removed from the top-level package.json. The dependencies on the top-level package.json should only be those which are required to build (e.g., turbo, commitzen, husky, @propeldata/release, etc.) and not babel, emotion, etc.

Reply:
We are now on Yarn PnP which should "hide" these, anyway.